### PR TITLE
Treaty of New York and Western Collapse

### PR DESCRIPTION
**Pull request by Perfo-6**

### CODE REVIEW

**Review by ColdSlav — COMMENTED** [review]
Looks good so far, please let me know of any other changes you haven't commited yet.

File: TGC/localisation/00_events_10000000.csv [hunk 1]
```diff
@@ -32,6 +32,9 @@ EVTOPTB31500001;The Union will continue with Estonian rule.;;;;;;;;;;;;;x
 EVTOPTC31500001;The Union will continue with the status quo.;;;;;;;;;;;;;x
 EVTOPTD31500001;Latvia will be released and we will declare Estonia free.;;;;;;;;;;;;;x
 EVTOPTE31500001;Estonia will be released and we will declare Latvia free.;;;;;;;;;;;;;x
+EVTDESC4395438;This event ensures that Pet�n is getting colonized.;;;;;;;;;;;x
```
Comment (ColdSlav) [inline TGC/localisation/00_events_10000000.csv:35]: Encoding error, please fix

**Review by ColdSlav — CHANGES_REQUESTED** [review]
Everything good, apart from one file with encoding error

File: TGC/localisation/00_events_10000000.csv (hunk 1 above)
Comment (GeneralVonGelre) [reply TGC/localisation/00_events_10000000.csv:35]: Its not.

**Review by GeneralVonGelre — COMMENTED** [review]
All a-ok..

**Comment by ColdSlav** [comment]
@GeneralVonGelre Are we good for merging this one?

**Review by ColdSlav — APPROVED** [review]
:+1:

**Pull request merged**